src/cryptography/hazmat/backends/interfaces.py

```diff
@@ -4,6 +4,19 @@
 
 
 import abc
+import typing
+
+from cryptography.hazmat._types import _PRIVATE_KEY_TYPES
+from cryptography.hazmat.primitives import hashes
+from cryptography.x509.base import Certificate
+from cryptography.x509.base import CertificateBuilder
+from cryptography.x509.base import CertificateRevocationList
+from cryptography.x509.base import CertificateRevocationListBuilder
+from cryptography.x509.base import CertificateSigningRequest
+from cryptography.x509.base import CertificateSigningRequestBuilder
+from cryptography.x509.base import RevokedCertificate
+from cryptography.x509.base import RevokedCertificateBuilder
+from cryptography.x509.name import Name
```

Comment:
We prefer to have a single `from cryptography.x509.base import (A, b, C, D)` with newlines.

Reply:
fixed